docs/core/compatibility/core-libraries/6.0/strong-name-signing-exceptions.md

```diff
@@ -31,8 +31,15 @@ Previously, an application that called the API compiled and ran, but as soon as
 
 Strong-name signing is not supported in .NET/.NET Core, and there is no workaround.
 
+> [!NOTE]
+> .NET Core/5+ never checks signatures in its runtime. However, if you're targeting cross-platform libraries (for example, a basic auth package that targets .NET Standard 2.0, so it runs on .NET Framework too), then strong-naming is a good idea for cross-runtime compatibility. .NET Framework continues to enforces strong naming if the calling app is strong-named. You can strong-name assemblies in all versions of .NET using the [Sn.exe](../../../../framework/tools/sn-exe-strong-name-tool.md) tool. For more information, see [Strong name signing](https://github.com/dotnet/runtime/blob/main/docs/project/strong-name-signing.md).
```

Comment:
```suggestion
> .NET Core/5+ never checks signatures in its runtime. However, if you're targeting cross-platform libraries (for example, a basic auth package that targets .NET Standard 2.0, so it runs on .NET Framework too), then strong-naming is a good idea for cross-runtime compatibility. The .NET Framework continues to enforce strong naming if the calling app is strong-named. You can strong-name assemblies in all versions of .NET using the [Sn.exe](../../../../framework/tools/sn-exe-strong-name-tool.md) tool. For more information, see [Strong name signing](https://github.com/dotnet/runtime/blob/main/docs/project/strong-name-signing.md).
```

Reply:
Thanks!

I didn't add "The" [.NET Framework] because that's actually incorrect - although I'm still trying to find the official guidance to point you to.

Reply:
Hi @gewarren - I still believe that there is a typo here, "enforces" should be "enforce", no?

Reply:
Yes, I updated that part. Thanks!